### DISCUSSION TITLE
Deploy Config: path

### DISCUSSION THREAD

**Discussion by ipsingh79** [Azure-Player/azure.datafactory.tools#155] [Q&A] [answered]
Hi @NowinskiK 
Quick question regarding the deploy config csv files, is the below row correct to refer the pipeline properties to replace during deployment? I am not very sure about the 'path' if that is the right way to add in the config.

type,name,path,value
pipeline,PL_SendNotification,activities[0].typeProperties.url,"https://prod-XXXX"

![image](https://user-images.githubusercontent.com/32059086/137175340-aab0e2c5-3162-4db5-b06f-60eb158eb844.png)

**Comment by NowinskiK** [comment, marked as answer]
Yes, but it's pretty risky to enumerate it like this, as likely it can be changed over time.
It is better to use the following pattern:
```
pipeline,PL_SendNotification,activities["TriggerNotificationLogicApp"].typeProperties.url,"https://prod-XXXX"
```
which is available in the latest version and documented here: [Column Path - example](https://github.com/SQLPlayer/azure.datafactory.tools#:~:text=pipeline%2CPL_Demo%2Cactivities%5B%22Copy%20Data%22%5D.typeProperties.waitTimeInSeconds%2C30)

But, **even better idea** is to use **Global Parameter** for this purpose.
